komorebi/bubblemenu/menu.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from typing import List
+
+from gi.repository import Clutter, GObject
+
+MENU_OPEN_SIGNAL = 'menu_open_event'
+MENU_CLOSE_SIGNAL = 'menu_close_event'
+
+class BubbleMenu(Clutter.Actor):
+	def __init__(self, screen):
+		Clutter.Actor.__init__(self)
```

Comment:
nitpick: Any reason you are not using `super()` here?

Reply:
I remember having some problems by using `super()` instead of the class name to call `__init__`, but can't remember exactly where and what. I'll test on the code again to see if these come up again.

Reply:
`super()` should return a proxy object of the parent, or sibling (in case of multiple inheritance), so it *should* work here.

Reply:
I have tested, and for some reason it doesn't work. From every bit of the official PyGTK/PyGObject documentation, they always call the parent constructor this way, so I assume this is how it's intended to be used.